core/prelude-packages.el

```diff
@@ -50,8 +50,13 @@
 
 ;; set package-user-dir to be relative to Prelude install path
 (setq package-user-dir (expand-file-name "elpa" prelude-dir))
-(package-initialize)
-
+;; Package initialize in emacs 27
+(if (or (version< emacs-version "27")
```

Comment:
Using `when` would produce better code here.

Reply:
I'm very new to elisp,
would this be better code?

```
when (or (version< emacs-version "27")
          (progn
           (not package--initialized)
           (setq package-quickstart t)))
(package-initialize))
```